src/pages/waiting-room-to-car/cat-be/components/vehicle-checks/vehicle-checks.html

```diff
@@ -6,10 +6,21 @@
   <ion-col align-self-center padding-left>
     <ion-row class="spacing-row"></ion-row>
     <ion-row>
-      <ion-col col-32>
+      <ion-col>
         <button type="button" class="mes-return-button" (click)="openVehicleChecksModal()" ion-button>
           <h3>Select questions</h3>
         </button>
+
+      </ion-col>
+      <ion-col class="vehicle-check-score">
+        <ion-row>
+          <serious-fault-badge class="badge" [showBadge]="true"></serious-fault-badge>
+          {{ vehicleChecksScore.seriousFaults }} serious fault
```

Comment:
should this display 'fault' or 'faults', depending on number?

Reply:
This will be handled in a separate PR